website/content/partials/builder/amazon/common/MetadataOptions-not-required.mdx

```diff
@@ -0,0 +1,10 @@
+<!-- Code generated from the comments of the MetadataOptions struct in builder/amazon/common/run_config.go; DO NOT EDIT MANUALLY -->
```

Comment:
You should these two newly generated docs files to the builders' documentation otherwise this won't be shown anywhere in the docs. 

All the amazon builders docs are https://github.com/hashicorp/packer/tree/master/website/content/docs/builders/amazon